File: 0080-remove-duplicates-from-sorted-array-ii/0080-remove-duplicates-from-sorted-array-ii.js
/**
 * @param {number[]} nums
 * @return {number}
 */
var removeDuplicates = function(nums) {
    if(nums.length <= 2)    return nums.length
    let k = 2
    for(var i=2; i<nums.length; i++){
        if(nums[i] != nums[k-2]){
            nums[k] = nums[i]
            k++
        }
    }
    return k
};

File: 0080-remove-duplicates-from-sorted-array-ii/0080-remove-duplicates-from-sorted-array-ii.cpp
class Solution {
public:
    int removeDuplicates(vector<int>& nums) {
        if(nums.size() == 1)    return 1;
        int curr = 2;
        for(int i=2; i<nums.size(); i++){
            if(nums[i] != nums[curr-2]){
                nums[curr] = nums[i];
                curr++;
            }
        }
        return curr;
    }
};